vignettes/parallel.Rmd

```diff
@@ -17,11 +17,23 @@ knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
 
 To take advantage of parallel tests, add the following line to the `DESCRIPTION`:
 
-    Config/testthat/parallel: true
+```         
+Config/testthat/parallel: true
+```
 
 You'll also need to be using the 3rd edition:
 
-    Config/testthat/edition: 3
+```         
+Config/testthat/edition: 3
+```
+
+By default, parallel testthat will use 2 cores.
+To customise that value for your local computer we recommend adding an entry for `TESTTHAT_CPUS` to your `.Rprofile`.
+The easiest way to do that is use `usethis::edit_r_profile()` and then add something like the following:
+
+```         
+TESTTHAT_CPUS=4
+```
```

Comment:
'Basic operation' right below already mentions some of this, so maybe the two paragraphs could be merged?